src/magic.rs

```diff
@@ -186,18 +185,6 @@ impl fmt::Debug for MagicEntry {
     }
 }
 
-impl Ord for MagicEntry {
```

Comment:
Yeah, that was mostly me seeing "oh, a priority, I bet we can use it", but after reading the spec it's pretty clear that priority is just not important within the magic vector; it's only used when implementing the checking algorithm, as a way to discriminate between two or more matching results.